src/components/application_manager/test/hmi_capabilities_test.cc

```diff
@@ -512,9 +512,6 @@ TEST_F(
   EXPECT_TRUE(navigation_capability_so.keyExists("getWayPointsEnabled"));
   EXPECT_TRUE(navigation_capability_so["sendLocationEnabled"].asBool());
   EXPECT_TRUE(navigation_capability_so["getWayPointsEnabled"].asBool());
-
-  // Since we have navigation capabilities, the feature should be supported
-  EXPECT_TRUE(hmi_capabilities_->navigation_supported());
```

Comment:
@RomanReznichenkoLuxoft why did you remove that expectation?

Reply:
done